Jenkinsfile

```diff
@@ -159,7 +159,7 @@ pipeline {
                     steps {
                         checkPatch user: GITHUB_USER_USR,
                                    password: GITHUB_USER_PSW,
-                                   ignored_files: "src/control/vendor/*:src/include/daos/*.pb-c.h:src/common/*.pb-c.[ch]:src/mgmt/*.pb-c.[ch]:src/iosrv/*.pb-c.[ch]:src/security/*.pb-c.[ch]:*.crt:*.pem"
+                                   ignored_files: "src/control/vendor/*:src/include/daos/*.pb-c.h:src/common/*.pb-c.[ch]:src/mgmt/*.pb-c.[ch]:src/iosrv/*.pb-c.[ch]:src/security/*.pb-c.[ch]:*.crt:*.pem:*_test.go"
```

Comment:
Yeah, I kinda like having the expected output be close to the literal representation though. It's easier to see what it's supposed to look like, and using cmp.Diff() on the strings makes it easy to see what's wrong when it differs.

For what it's worth, I used this format to iterate over tabwriter options until I found the right combination that gave me the output I was expecting to see.

Reply:
That's fair, I get it. Though I thought of another reason to spell the whitespace characters out literally (or remove the trailing tabs from the output): It would be really easy to strip the trailing tabs from a test file by accident, just by using an editor set up to strip trailing whitespace automatically.